.github/workflows/sp-deployment-pipeline.yml

```diff
@@ -0,0 +1,56 @@
+name: Deploy Sample Platform
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+
+# permission can be added at job level or workflow level
```

Comment:
Please remove the extra comments that are "What" in nature and not "Why".

Also, please add permission on the job level, just to be more secure.

Reply:
Sure sure